azkaban-common/src/main/java/azkaban/executor/Executor.java

```diff
@@ -28,8 +29,9 @@
   private final String host;
   private final int port;
   private boolean isActive;
-
-  // TODO: ExecutorStats to be added
+  // cached copy of the latest statistics from  the executor.
```

Comment:
I was actually thinking of having the Statistics class below named ExecutorInfo or ExecutorStats..